sign_language_datasets/datasets/chicagofswild/chicagofswild.py

```diff
@@ -39,14 +43,20 @@
 _CHICAGO_FS_WILD_PLUS_URL = "https://dl.ttic.edu/ChicagoFSWildPlus.tgz"
 _CHICAGO_FS_WILD_URL = "https://dl.ttic.edu/ChicagoFSWild.tgz"
 
+_POSE_URLS = {"holistic": "/shares/volk.cl.uzh/zifjia/ChicagoFSWild/pose/"}
+_POSE_HEADERS = {"holistic": path.join(path.dirname(path.realpath(__file__)), "holistic.poseheader")}
```

Comment:
@AmitMY I just copied a `holistic.poseheader` from another dataset, not sure how it works ...

Reply:
Depends on which dataset - easiest solution is:
```py
with open(".../...pose", "rb") as f:
  pose = Pose.read(f.read())
with open("holistic.poseheader", "wb") as f:
  pose.header.write(f)
```

(written in the comment, might contain an error)

Reply:
Okay this is implicit, why/when is this header needed?

Reply:
the dataset loader basically gives you a `PoseBody`, but no information about it (which is the header)
so if you want to normalize the pose, or perform any high level operation (visualization) you need to know the header